examples/jenkins/jenkins-ephemeral-template.json

```diff
@@ -136,19 +146,42 @@
       }
     },
     {
+      "kind": "ServiceAccount",
+        "apiVersion": "v1",
+        "metadata": {
+            "name": "${JENKINS_SERVICE_NAME}"
+        }
+    },
+    {
+      "kind": "RoleBinding",
+      "apiVersion": "v1",
+      "metadata": {
+          "name": "${JENKINS_SERVICE_NAME}_edit"
```

Comment:
we've been doing this in the jenkins template (the one we auto instantiate) basically from day one.  the only change here is that now we're doing it in the example template that we expect people to instantiate themselves.  So if anything, this is better since they're taking an explicit action to do it.